controller.go

```diff
@@ -253,14 +302,30 @@ func (c *controller) NewNetwork(networkType, name string, options ...NetworkOpti
 
 	network.processOptions(options...)
 
-	if err := c.addNetwork(network); err != nil {
+	if _, err := c.loadNetworkDriver(network); err != nil {
```

Comment:
The plugin does not get loaded twice, because the driver would be present in `c.drivers` when the second call is executed.